certified-connectors/Payables E2E/apiDefinition.swagger.json

```diff
@@ -0,0 +1,132 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "IN-D Invoice Data Capture",
+    "description": "A smart invoice processing solution that converts static documents (scanned, PDFs, screenshots, etc.) into usable information to fuel your business processes.IN-D Payables is an AI led intelligent document classification and capture solution that automates processing of all kinds of commercial documents used in Shipping, Logistics and Manufacturing operations. Manual handling of documents slows the pace of business transactions."
+  },
+  "host": "payables-sit.in-d.ai",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/api/users/token/": {},
+    "/api/e2e": {
+      "post": {
+        "summary": "Payable E2E",
+        "description": "Process invoices to extract the key attributes like invoice numbers, PO number, vendor name, etc., and export the data",
+        "operationId": "Payable_E2E",
```

Comment:
@gayathri35 operationId properties [should be in pascal case](https://docs.microsoft.com/en-us/connectors/custom-connectors/coding-standards#actions-and-triggers) and without hyphens or underscores.